pyproject.toml

```diff
@@ -0,0 +1,40 @@
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [ "setuptools>=45", "setuptools_scm[toml]>=6.2",]
+
+[project]
+classifiers = [ "Development Status :: 2 - Pre-Alpha", "Natural Language :: English", "Programming Language :: Python :: 3",]
```

Comment:
I think this app has to be at least `Development Status :: 5 - Production/Stable`, right? It's in constant use in production.